src/ucs/datastruct/strided_alloc.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2018.  ALL RIGHTS RESERVED.
```

Comment:
AFAIR, for new files just one date needs to be specified (2018). Relevant for other new files as well.

Reply:
ok